### DISCUSSION TITLE
Python Package support for OpenAI's Certificate

### DISCUSSION THREAD

**Discussion by endamaco** [open-webui/open-webui#4293] [General]
# Bug Report

## Description

**Bug Summary:**
I've installed the python package
`pip install open-webui`
`open-webui serve`
But when I enter the OpenAI's key and try to connect I can see the following error in the console

> ERROR:apps.openai.main:Connection error: Cannot connect to host api.openai.com:443 ssl:True [SSLCertVerificationError: (1, '[SSL: CERTIFICATE_VERIFY_FAILED] certificate verify failed: unable to get local issuer certificate (_ssl.c:992)')]

**Steps to Reproduce:**
Open WebUI (manual python, not docker). Go in admin panel -> settings -> connections -> enter OpenAI API's key and click "Verify Connection"

**Expected Behavior:**
Connect to OpenAI's endpoint

**Actual Behavior:**
An error with SSL certificates prevent it to work. If I open the OpenAI's endpoint in a browser I don't have any issue with certificates

## Environment

- **Open WebUI Version:** 0.3.10
- **Ollama (if applicable):** N/A

- **Operating System:** MacOS
- **Browser (if applicable):** Chrome

## Installation Method

Manual Installation

**Comment by ssuriyayuvan** [comment]
same problem for me. Did you get any solution for this?
